components/Avatars/Avatar.css

```diff
@@ -0,0 +1,90 @@
+.header{
+  display: flex;
+   padding: 0.5rem 4rem;
+   justify-content: space-between;
+   align-items: center;
+   box-shadow: 2px 5px 8px grey;
```

Comment:
make different variables for box shadows with increasing intensity
it will help to maintain consistency

Reply:
okay..I will make it..Thanks